src/Sarif.Viewer.VisualStudio/Sarif.Viewer.VisualStudio.csproj

```diff
@@ -374,15 +383,6 @@
       <Generator>MSBuild:Compile</Generator>
     </Page>
   </ItemGroup>
-  <ItemGroup />
-  <PropertyGroup>
```

Comment:
There was an issue with PREfastXmlSarifConverter.dll not being available for copy at the correct time.  Removed this and now included a .dll in a /lib folder